crates/api_models/src/admin.rs

```diff
@@ -1934,6 +1934,9 @@ pub struct BusinessProfileCreate {
     /// These key-value pairs are sent as additional custom headers in the outgoing webhook request. It is recommended not to use more than four key-value pairs.
     #[schema(value_type = Option<Object>, example = r#"{ "key1": "value-1", "key2": "value-2" }"#)]
     pub outgoing_webhook_custom_http_headers: Option<HashMap<String, String>>,
+
+    /// Merchant Connector id to be stored for tax_calculator connector
+    pub tax_connector_id: Option<String>,
```

Comment:
Taking tax_connector_id here would force us to update the business profile every time a new MCA is added. Instead, taking connector_name here would avoid this friction

Reply:
cc: @SamraatBansal

Reply:
Updating of business profile is required to map to specific tax mca, so id is fine, name we can infer, but in case of name we will not be able to choose specific tax mca